src/order/hom/bounded.lean

```diff
@@ -289,6 +289,13 @@ lemma cancel_left {g : bot_hom β γ} {f₁ f₂ : bot_hom α β} (hg : injectiv
 ⟨λ h, bot_hom.ext $ λ a, hg $
   by rw [←bot_hom.comp_apply, h, bot_hom.comp_apply], congr_arg _⟩
 
+/-- Reinterpret a bot homomorphism as a top homomorphism between the dual lattices. -/
+@[simps] protected def dual : bot_hom α β ≃ top_hom (order_dual α) (order_dual β) :=
+{ to_fun := λ f, ⟨f, f.map_bot'⟩,
+  inv_fun := λ f, ⟨f, f.map_top'⟩,
+  left_inv := λ f, bot_hom.ext $ λ _, rfl,
+  right_inv := λ f, top_hom.ext $ λ _, rfl }
+
```

Comment:
Could you add that these maps preserve `id` and `comp` both ways? (Ideally we should be able to state that these maps are an isomorphism of categories, but we don't quite have the machinery for generic `id`/`comp`.)

Reply:
An isomorphism of categories? Definitely not why I'm defining those! 😝

Reply:
Do you really want the other way? I find those a bit odd but maybe I'm wrong.

Reply:
By "the other way" I meant `dual.symm id = id`, which I don't find very weird. (Of course, it's just a `symm_apply_eq` away from the ones you added so if you still object I'm ok with that.)